app/scripts/views/components/dataTable/tableViewTemplate.js

```diff
@@ -83,10 +84,12 @@
                 'patient_uid' : 'sample_uid';
             var _selectedCases =
               _.pluck(this.invisibleDimension.top(Infinity), attrId);
-            this.chartInst.update(_selectedCases, this.selectedRows);
-            this.setDisplayTitle(this.chartInst.getCases().length);
-            this.showLoad = false;
-            this.showRainbowSurvival();
+            if (this.dataLoaded) {
```

Comment:
if (!this.failedToInit) {
          if (this.madeSelection && !this.isMutatedGeneCna) {
            this.madeSelection = false;
          } else if (this.dataLoaded) {
            var attrId =
              this.attributes.group_type === 'patient' ?
                'patient_uid' : 'sample_uid';
            var _selectedCases =
              _.pluck(this.invisibleDimension.top(Infinity), attrId);
            this.chartInst.update(_selectedCases, this.selectedRows);
            this.setDisplayTitle(this.chartInst.getCases().length);
            this.showLoad = false;
            this.showRainbowSurvival();
          }
    }